src/app/shared/pipes/app-panels-add-aria-label/app-panels-add-aria-label.pipe.ts

```diff
@@ -0,0 +1,24 @@
+import { Pipe, PipeTransform } from '@angular/core'
+
+@Pipe({
+  name: 'appPanelsAddAriaLabel',
+})
+export class AppPanelsAddAriaLabelPipe implements PipeTransform {
+  transform(panelType: any, args?: any): string {
+    if (panelType == 'employment') {
```

Comment:
We still have a few `==` comparators, I will soon enable EsLint so all these issues are automatically flagged for developers.

Reply:
Solved. Hopefully all of them :)